### PR TITLE
Adding support for histograms

### PR DESCRIPTION
**Pull request by lcavadas**
- Added a new HttpHistogramMeterFilterFactory class to contribute to the configuration of the 'http.server.requests' and 'http.client.requests' meters.

- Added test to validate configuration takes effect.

### CODE REVIEW

File: micrometer-core/src/main/java/io/micronaut/configuration/metrics/binder/web/HttpPercentilesMeterFilterFactory.java [hunk 1]
```diff
@@ -39,43 +37,46 @@
 @Factory
 @RequiresMetrics
 @Requires(property = WebMetricsPublisher.ENABLED, notEquals = FALSE)
-public class HttpMeterFilterFactory {
+public class HttpPercentilesMeterFilterFactory {
```
Comment (sdelamo) [inline micrometer-core/src/main/java/io/micronaut/configuration/metrics/binder/web/HttpPercentilesMeterFilterFactory.java]: This is a breaking change. We cannot rename public classes in a patch version.

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: micrometer-core/src/main/java/io/micronaut/configuration/metrics/binder/web/HttpPercentilesMeterFilterFactory.java (hunk 1 above)
Comment (lcavadas) [reply micrometer-core/src/main/java/io/micronaut/configuration/metrics/binder/web/HttpPercentilesMeterFilterFactory.java]: Done, classname back to original 👍

**Review by graemerocher — APPROVED** [review]

**Comment by graemerocher** [comment]
Thanks for the contribution!

**Review by scprek — COMMENTED** [review]
nvm docs are updated.

**Pull request merged**